include/experimental/__p0009_bits/macros.hpp

```diff
@@ -18,7 +18,12 @@
 
 #include "config.hpp"
 
+#include <cstdio>
+#include <cstdlib>
 #include <type_traits> // std::is_void
+#if defined(_MDSPAN_HAS_CUDA) || defined(_MDSPAN_HAS_HIP) || defined(_MDSPAN_HAS_SYCL)
+#include "assert.h"
```

Comment:
Please confirm that you mean to include this header and not `<cassert>`

Reply:
I couldn't find a combo which fails. For Kokkos we anyway gonna replace the thing with Kokkos::abort.